db/migrate/20190116005251_seed_metadata_fields.rb

```diff
@@ -0,0 +1,416 @@
+    # Ex: lat_lon could be a core field but not on new projects by default.
+    # t.integer :is_core
+
+    # Default fields are a subset of the core fields that will appear on a project when someone
+    # creates a project. These can be removed from a project.
+    # t.integer :is_default
+
+    # Required fields are a subset of the core/default fields that cannot be removed from the
+    # project. This is the strictest level. Ex: collection_date, location, nucleotide_type, etc.
+    # t.integer :is_required
+
+    to_create = []
+
+    ############################
+    ##### ALL HOST GENOMES #####
+    ############################
+
+    to_create << MetadataField.new(
+      name: "sample_type",
+      display_name: "Sample Type",
+      description: "Type of sample or tissue",
+      base_type: Metadatum::STRING_TYPE,
+      is_core: 1,
+      is_default: 1,
+      is_required: 1,
+      group: "Sample",
+      host_genomes: HostGenome.all
+    )
+
+    to_create << MetadataField.new(
+      name: "nucleotide_type",
+      display_name: "Nucleotide Type",
+      description: "DNA or RNA",
+      base_type: Metadatum::STRING_TYPE,
+      options: %w[DNA RNA],
+      force_options: 1,
+      is_core: 1,
+      is_default: 1,
+      is_required: 1,
+      group: "Sample",
+      host_genomes: HostGenome.all
+    )
+
+    to_create << MetadataField.new(
+      name: "collection_date",
+      display_name: "Collection Date",
+      description: "Date sample was originally collected",
+      base_type: Metadatum::DATE_TYPE,
+      is_core: 1,
+      is_default: 1,
+      is_required: 1,
+      group: "Sample",
+      host_genomes: HostGenome.all
+    )
+
+    to_create << MetadataField.new(
+      name: "collection_location",
+      display_name: "Collection Location",
+      description: "Location sample was originally collected",
+      base_type: Metadatum::STRING_TYPE,
+      is_core: 1,
+      is_default: 1,
+      is_required: 1,
+      group: "Sample",
+      host_genomes: HostGenome.all
+    )
+
+    to_create << MetadataField.new(
+      name: "collected_by",
+      display_name: "Collected By",
+      description: "Collecting institution/agency",
+      base_type: Metadatum::STRING_TYPE,
+      is_core: 1,
+      group: "Sample",
+      host_genomes: HostGenome.all
+    )
+
+    to_create << MetadataField.new(
+      name: "sex",
+      display_name: "Sex",
+      description: "Sex of the host/participant/specimen",
+      base_type: Metadatum::STRING_TYPE,
+      options: %w[Female Male],
+      force_options: 1,
+      is_core: 1,
+      is_default: 1,
+      group: "Host",
+      host_genomes: HostGenome.all
+    )
+
+    to_create << MetadataField.new(
+      name: "known_organism",
+      display_name: "Known Organism",
+      description: "Organism detected by a clinical lab",
+      base_type: Metadatum::STRING_TYPE,
+      is_core: 1,
+      is_default: 1,
+      group: "Host",
```

Comment:
Comorbidity, Infection Class, Known Organism, and Detection Method should belong to the "Infection" group.